### DISCUSSION TITLE
Problem with post publications

### DISCUSSION THREAD

**Discussion by chiaraepoi** [Automattic/wordpress-activitypub#537] [General]
I have two accounts, one on a Mastodon istance, another on a Pleroma instance and a blog self hosted with ActivityPub plugin installed.
When I write a post on my blog, publication is ok on the Pleroma server, but same post not appears in the Mastodon instance.
Someone know why?
Tanks

**Comment by pfefferle** [comment]
Can you tell us the mastodon instance?

**Comment by chiaraepoi** [comment]
https://bologna.one

**Reply by pfefferle** [reply]
is it possible to follow your blog on bologna.one? Just to be sure if it is a follow issue or an issue with the post-federation.

**Comment by chiaraepoi** [comment]
The account of the blog is @chiara@www.ilblogdichiara.it
My account on the mastodon instance bologna.one  is @chiaraepoi@bologna.one

**Reply by pfefferle** [reply]
Ok, it seems that "following" does not work. Have you installed a caching plugin on your blog? If so, can you try to deactivate it and re-try following your profile?

**Reply by chiaraepoi** [reply]
Yes I have a caching plugin. I try to deactivate and retry

**Reply by chiaraepoi** [reply]
I deactivate th cache plugin, but nothing changes.
Today the post doesn't even appear in the pleroma instance

**Reply by pfefferle** [reply]
it worked for me (on mastodon.social). maybe it is cached by your mastodon server, can you try to search for `chiara@www.ilblogdichiara.it` or `https://www.ilblogdichiara.it/author/chiara/`. If you want to re-activate your caching plugin, you have to check if it is possible to exclude the author pages from the cache `https://www.ilblogdichiara.it/author/chiara/`.

**Reply by pfefferle** [reply]
@chiaraepoi that the post do not appear on pleroma and mastodon is a coincidence with the caching plugin. try to update the post with disabled caching (or excluded author profiles) and it should appear on both platforms.

**Reply by pfefferle** [reply]
(I just received your lore-ipsum post on mastodon btw. 😉)

**Comment by chiaraepoi** [comment]
I made a mistake in my previous test. I have now done the test correctly and everything is ok on the pleroma. But I still don't see the post on Mastodon

**Reply by pfefferle** [reply]
have you re-followed your profile?

**Reply by pfefferle** [reply]
btw. you do not have to create a new blog-post to test, you could simply update an old one.

**Reply by chiaraepoi** [reply]
I asked two friends with accounts on other mastodon instances to follow the blog account and I discovered that on some instances the post is not visible (bologna.one, livellosegreto.it) while on others (mastodon.uno, for example) it reads correctly

**Reply by pfefferle** [reply]
can you try updating one of your older posts? maybe this one? https://www.ilblogdichiara.it/chi-lavrebbe-detto/

**Reply by chiaraepoi** [reply]
Done...

**Reply by pfefferle** [reply]
it seems that you are not listed as a follower of your blog, can you try to re-follow before updating a post. https://www.ilblogdichiara.it/wp-json/activitypub/1.0/users/2/followers?page=1
